setup.py

```diff
@@ -19,8 +19,22 @@
 from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 from version import version
+from pkg_resources import parse_version
 
-toil_version = '3.1.4'
+# Toil version check -- Raise warning instead of using intall_requires to avoid virtualenv conflicts
+toil_minor_version = 3.1
```

Comment:
Wow, that's good to know. When can we switch over to [Unums](http://www.johngustafson.net/presentations/Unums2.0.pdf)? 

This new `parse_version` object has a method that returns the version, base_version, which just returns a string of '3.1', which sort of puts me back at square one.